src/main/java/com/dreammaster/gthandler/recipes/CircuitAssemblerRecipes.java

```diff
@@ -66,391 +66,409 @@ public void run() {
+                .addTo(sCircuitAssemblerRecipes);
+
+        GT_Values.RA.stdBuilder()
+                .itemInputs(
+                        ItemList.Circuit_Board_Multifiberglass_Elite.get(1L),
+                        ItemList.Circuit_Crystalcomputer.get(2L),
+                        ItemList.Circuit_Chip_Ram.get(4L),
+                        ItemList.Circuit_Chip_NOR.get(32L),
+                        ItemList.Circuit_Chip_NAND.get(64L),
+                        GT_OreDictUnificator.get(OrePrefixes.wireFine, Materials.NiobiumTitanium, 32))
+                .itemOutputs(ItemList.Circuit_Ultimatecrystalcomputer.get(1L))
+                .fluidInputs(new FluidStack(solderIndalloy, 144)).noFluidOutputs().requiresCleanRoom()
+                .duration(20 * SECONDS).eut(TierEU.RECIPE_IV).addTo(sCircuitAssemblerRecipes);
+
+        GT_Values.RA.stdBuilder()
+                .itemInputs(
+                        GT_OreDictUnificator.get(OrePrefixes.frameGt, Materials.Aluminium, 2),
+                        ItemList.Circuit_Ultimatecrystalcomputer.get(2L),
+                        ItemList.Circuit_Parts_InductorASMD.get(8L),
+                        ItemList.Circuit_Parts_CapacitorASMD.get(16L),
+                        ItemList.Circuit_Chip_Ram.get(32L),
+                        GT_OreDictUnificator.get(OrePrefixes.wireGt01, Materials.SuperconductorLuV, 16))
+                .itemOutputs(ItemList.Circuit_Crystalmainframe.get(1L)).fluidInputs(new FluidStack(solderIndalloy, 288))
+                .noFluidOutputs().requiresCleanRoom().duration(40 * SECONDS).eut(TierEU.RECIPE_LuV)
+                .addTo(sCircuitAssemblerRecipes);
         // Wetware Circuits
-        GT_Values.RA.addCircuitAssemblerRecipe(
-                new ItemStack[] { ItemList.Circuit_Chip_NeuroCPU.get(1L), ItemList.Circuit_Chip_CrystalCPU.get(1L),
-                        ItemList.Circuit_Chip_NanoCPU.get(1L), ItemList.Circuit_Parts_CapacitorASMD.get(8L),
+
+        GT_Values.RA.stdBuilder()
+                .itemInputs(
+                        ItemList.Circuit_Chip_NeuroCPU.get(1L),
+                        ItemList.Circuit_Chip_CrystalCPU.get(1L),
+                        ItemList.Circuit_Chip_NanoCPU.get(1L),
+                        ItemList.Circuit_Parts_CapacitorASMD.get(8L),
                         ItemList.Circuit_Parts_TransistorASMD.get(8L),
-                        GT_OreDictUnificator.get(OrePrefixes.wireFine, Materials.YttriumBariumCuprate, 8) },
-                new
-
-                FluidStack(solderIndalloy, 72),
-                ItemList.Circuit_Neuroprocessor.get(1L),
-                200,
-                38400,
-                true);
+                        GT_OreDictUnificator.get(OrePrefixes.wireFine, Materials.YttriumBariumCuprate, 8))
+                .itemOutputs(ItemList.Circuit_Neuroprocessor.get(1L)).fluidInputs(new FluidStack(solderIndalloy, 72))
+                .noFluidOutputs().requiresCleanRoom().duration(10 * SECONDS).eut(TierEU.RECIPE_LuV)
```

Comment:
wrong voltage

Reply:
well and so on. most but not all are one voltage too low. it was like 38000 (so zpm) and you make it TierEU.RECIPE_LuV